tests/utils/exec/machine-pools.go

```diff
@@ -114,6 +119,21 @@ func NewMachinePoolService(manifestDir ...string) *MachinePoolService {
 	return mp
 }
 
+// ****************************** Machinepool functions ***************************
+
+func RetrieveClusterMachinePoolResponseBody(connection *client.Connection, clusterID string, name string) (*cmv1.MachinePool, error) {
```

Comment:
@xueli181114 Danielle suggested to move this to different function as the number of these functions are used miltiple times in the TCs. Do you think it is right place to put?

Reply:
Actually this is functions used for our verification. Let's not put it under exec.  Let's put it to `terraform-provider-rhcs/tests/utils/cms`

Reply:
done after discussing in chat